### PR TITLE
Implement Artifact Details Page

### LINKED ISSUE #11: Implement Screen for Detailed View of Artifacts and Related Artifacts
We need to create a Screen that displays a detailed view of the artifact and the list of related artifacts. 

The Screen should show details about the artifact and provide a section for a list of related artifacts. Users should be able to click on related artifacts to view their details.

### PR DESCRIPTION
**Pull request by puru-khedre**
Closes: #11

### CODE REVIEW

File: screen/ArtifactLifeCycleApp.xml [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+public domain worldwide. This software is distributed without any
+warranty.
+
+You should have received a copy of the CC0 Public Domain Dedication
+along with this software (see the LICENSE.md file). If not, see
+<http://creativecommons.org/publicdomain/zero/1.0/>.
+-->
+<screen xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:noNamespaceSchemaLocation="http://moqui.org/xsd/xml-screen-3.xsd"
+        default-menu-title="nit"
```
Comment (puru-khedre) [inline screen/ArtifactLifeCycleApp.xml]: What is nit

File: screen/ArtifactLifeCycleApp/ArtifactEvent/ViewArtifactEvent.xml [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+along with this software (see the LICENSE.md file). If not, see
+<http://creativecommons.org/publicdomain/zero/1.0/>.
+-->
+
+<screen xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:noNamespaceSchemaLocation="http://moqui.org/xsd/xml-screen-3.xsd"
+>
+    <parameter name="artifactEventId" required="true"/>
+
+    <transition name="viewArtifact">
```
Comment (puru-khedre) [inline screen/ArtifactLifeCycleApp/ArtifactEvent/ViewArtifactEvent.xml]: Rename to viewArtifactEvent

File: screen/ArtifactLifeCycleApp/ArtifactEvent/ViewArtifactEvent.xml [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+                <container-box>
+                    <box-header title="Life Cycle"/>
+                    <box-body></box-body>
+                </container-box>
+            </row-col>
+        </container-row>
+        <container-row>
+            <row-col sm="12">
+                <container-box>
+                    <box-header title="Related Artifact Log Events"/>
```
Comment (puru-khedre) [inline screen/ArtifactLifeCycleApp/ArtifactEvent/ViewArtifactEvent.xml]: Related Artifact Events

File: service/co/hotwax/alc/SearchService.xml [hunk 4]
```diff
@@ -0,0 +1,115 @@
...
+To the extent possible under law, the author(s) have dedicated all
+copyright and related and neighboring rights to this software to the
+public domain worldwide. This software is distributed without any
+warranty.
+
+You should have received a copy of the CC0 Public Domain Dedication
+along with this software (see the LICENSE.md file). If not, see
+<http://creativecommons.org/publicdomain/zero/1.0/>.
+-->
+<services xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="http://moqui.org/xsd/service-definition-3.xsd">
```
Comment (puru-khedre) [inline service/co/hotwax/alc/SearchService.xml]: Add comments and description

File: service/co/hotwax/alc/SearchService.xml
Comment (puru-khedre) [inline service/co/hotwax/alc/SearchService.xml:1]: Rename service name file

**Comment by puru-khedre** [comment]
@dt2patel please review this PR

**Review by dt2patel — APPROVED** [review]

**Pull request merged**